src/OrbitCore/SymbolHelper.cpp

```diff
@@ -116,7 +141,14 @@ ErrorMessageOr<fs::path> SymbolHelper::FindSymbolsWithSymbolsPathFile(
 
   LOG("Trying to find symbols for module: \"%s\"", module_path.string());
   for (const auto& symbols_path : search_paths) {
-    if (!fs::exists(symbols_path)) continue;
+    std::error_code error;
+    bool exists = fs::exists(symbols_path, error);
+    if (error) {
+      ERROR("Unable to stat \"%s\": %s", symbols_path.string(), error.message());
+      continue;
+    }
+
+    if (!exists) continue;
```

Comment:
Should we warn here?

Reply:
I believe this is perfectly fine since this method basically check the list which is constructed without knowledge of which files exists and which do not. it may contain for example "/path1/liba.so.debug", "/path2/lib.so.debug" if one of these exists symbol will be found. it is ok for both of them to not exists because in this case the caller will try to download them from the service or do something else.